### PR TITLE
filter "control-plane" labeled namespaces out of our webhooks

### PR DESCRIPTION
**Pull request by n3wscott**
fixes: #1590

### CODE REVIEW

File: webhook/resourcesemantics/defaulting/defaulting.go [hunk 1]
```diff
@@ -186,6 +186,9 @@ func (ac *reconciler) reconcileMutatingWebhook(ctx context.Context, caCert []byt
 			MatchExpressions: []metav1.LabelSelectorRequirement{{
 				Key:      "webhooks.knative.dev/exclude",
 				Operator: metav1.LabelSelectorOpDoesNotExist,
+			},{
```
Comment (evankanderson) [inline webhook/resourcesemantics/defaulting/defaulting.go:189]: Can we add a comment about where the `control-plane` annotation comes from (AKS), just for posterity?

Comment (dprotaso) [reply webhook/resourcesemantics/defaulting/defaulting.go:189]: Maybe link this upstream issue as well
https://github.com/Azure/AKS/issues/1771

**Pull request merged**